modules/virt-mounting-windows-driver-disk-on-vm.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-mounting-windows-driver-disk-on-vm_{context}"]
+
+= Mounting a Windows driver disk on a virtual machine
+
+You can mount a Windows driver disk on a virtual machine (VM) by using the {product-title} web console.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines*.
```

Comment:
@max-cx 
Yes, one could use 'Go to'. But since in this case it implies going through a number of pages and tabs, 'navigate' looks appropriate to me. I don't think the length of the words should be a consideration here.